tools/pipelines/build-azure.yml

```diff
@@ -49,6 +49,14 @@ trigger:
     - tools/pipelines/templates/include-publish-npm-package-deployment.yml
     - tools/pipelines/templates/include-publish-npm-package-steps.yml
     - tools/pipelines/templates/include-git-tag-steps.yml
+    # Relevant to the test-azure-frs pipeline
+    - packages/dds/map
+    - packages/drivers/routerlicious-driver
+    - packages/framework/fluid-static
+    - packages/framework/loader/container-loader
+    - packages/test/test-end-to-end-tests
+    - tools/pipelines/test-azure-frs.yml
+    - tools/pipelines/templates/include-test-real-service.yml
```

Comment:
Moved this over from the CI trigger removed from the test-azure-frs pipeline, so changes that would have triggered it independently now trigger Build - azure and in turn that triggers test-azure-frs. Added `packages/test/test-end-to-end-tests` which test-azure-frs uses.